src/component/sidebaritem/index.tsx

```diff
@@ -102,6 +104,19 @@ function SidebarItem(
             setDropdownVisible(false);
           },
         }}
+        hasNewActivity={unreadCount > 0}
+        counterSlot={{
+          children: (
+            <span
+              style={{
+                color: "#fff",
+                fontWeight: 500,
+              }}
+            >
+              {unreadCount}
+            </span>
+          ),
+        }}
```

Comment:
Will improve by changing counterSlot prop to type `number`

Reply:
Why can't we use the default Plasmic style here? Does it not look right?

Reply:
counterSlot prop actually expects an element, I tried with `counterSlot={unreadCount}` but the style doesn't match. Maybe we need to update on Plasmic?

Reply:
Makes sense. We can go with this for now then.